ArduPlane/tailsitter.cpp

```diff
@@ -142,28 +189,73 @@ void QuadPlane::tailsitter_output(void)
             motors->limit.roll_pitch = 1;
             motors->limit.yaw = 1;
         }
-        SRV_Channels::set_output_scaled(SRV_Channel::k_tiltMotorLeft, tilt_left);
-        SRV_Channels::set_output_scaled(SRV_Channel::k_tiltMotorRight, tilt_right);
     }
 
+    float aspeed;
+    bool have_airspeed = ahrs.airspeed_estimate(&aspeed);
+    const float scaling_range = tailsitter.scaling_speed_max - tailsitter.scaling_speed_min;
+    if (aspeed > tailsitter.scaling_speed_min && have_airspeed && !is_zero(scaling_range)) {
+        // apply surface scaling to interpolate between fixed wing and VTOL outputs based on airspeed
+        // kd0aij note: that the airspeed estimate based only on GPS and (estimated) wind is
+        // not sufficiently accurate for tailsitters. (based on tests in RealFlight 8 with 10kph wind)
```

Comment:
this is true, unless there is a airspeed sensor fitted, I guess we could check there is a actual airspeed sensor, but it is this airspeed that is used for control surface scaling in plane (i think).

Reply:
I figured the user would set spdmin=spdmax if there is no airspeed sensor